evm/chains/sharding/mainchain_handler/config.py

```diff
@@ -0,0 +1,9 @@
+# FIXME: should be 25 or 2500, fix it, only for faster testing
```

Comment:
This suggests that this  should be a default that's can be overridden in the places it's used.

Reply:
Get it. Since `SHUFFLING_CYCLE_LENGTH` is going to be deprecated in the new validator manager contract, I moved this to the test for explicit specification. Is it okay?